runelite-client/src/main/java/net/runelite/client/plugins/itemcharges/ItemChargePlugin.java

```diff
@@ -342,6 +343,24 @@ else if (message.equals(RING_OF_FORGING_BREAK_TEXT))
 
 				updateRingOfForgingCharges(MAX_RING_OF_FORGING_CHARGES);
 			}
+
+			else if (message.equals(RING_OF_FORGING_VARROCK_PLATEBODY))
```

Comment:
Since this is largely duplicated from L321 I think you can just add it there using eg 

```java
int charges = Ints.constrainToRange(
  getItemCharges(ItemChargeConfig.KEY_RING_OF_FORGING) - message.equals(RING_OF_FORGING_VARROCK_PLATEBODY) ? 2 : 1,
  0, MAX_RING_OF_FORGING_CHARGES);
```